Source/Shared/Storage/DiskStorage.swift

```diff
@@ -276,16 +276,7 @@ public final class DiskStorage: StorageAware {
    - Returns: A md5 or base64 string
    */
   func fileName(_ key: String) -> String {
-    if let digest = key.data(using: String.Encoding.utf8)?.md5() {
-      var string = ""
-      for byte in digest {
-        string += String(format:"%02x", byte)
-      }
-
-      return string
-    } else {
-      return key.base64()
```

Comment:
If it sometimes can't `md5`, then we fallback to `base64`, which will cause inconsistent